src/node_dotenv.cc

```diff
@@ -26,6 +27,33 @@ std::vector<Dotenv::env_file_data> Dotenv::GetDataFromArgs(
            arg.starts_with("--env-file-if-exists=");
   };
 
+  const auto get_sections = [](const std::string& path) {
```

Comment:
I recommend always using std::string_view since you'll always know when you'll copy (by calling std::string(val))

```suggestion
  const auto get_sections = [](std::string_view path) {
```